src/components/PersonName/PersonName.tsx

```diff
@@ -0,0 +1,19 @@
+// PersonName.jsx
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Sex } from '../../types/Sex';
+import { Person } from '../../types';
+
+export const PersonName = ({ name, slug, sex }: Partial<Person>) => {
```

Comment:
This is a `PersonLink` and it could accept the whole person.
- Make each name in the table a link a to /people/:slug (including mother and father)
![image](https://github.com/mate-academy/react_people-table-basics/assets/63243136/693648d3-6c09-4468-9154-0765670916e9)